fingerprint_verification.py

```diff
@@ -6,39 +6,103 @@
 
 def read_print(filename):
     """Reads the given print and returns a dictionary with all info."""
-    # Your turn!
-
-
-
-
-
-
-# You can add your various check functions below
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    fingerprint_data = {}
+    with open(filename) as fh:
+        name = fh.readline().strip()
+        width = int(fh.readline().strip())
+        height = int(fh.readline().strip())
+        
+        fingerprint_data['name'] = name
+        fingerprint_data['width'] = width
+        fingerprint_data['height'] = height
+
+        fingerprint = [] 
+
+        for line in filename:
```

Comment:
Wait, you are looping over the filename, not the file handle. This is just looping over the string that is the filename, not the actual contents of the files.